### PR TITLE
Translate interface into Bulgarian. България на три морета! 🇧🇬

### PR DESCRIPTION
**Pull request by suricactus**
@m-kuhn There are quite a lot of strings that are missing from the `*.ts` files and I had to run `scripts/ci/update-translations.sh` manually to extract them. Do I miss something or it this is the procedure to follow?

### CODE REVIEW

**Comment by m-kuhn** [comment]
Thanks :bulgaria: 
Strings are managed on transifex, I have uploaded the file there and added you as a language coordinator, upcoming dev builds should have the language https://www.transifex.com/opengisch/qfield-for-qgis/translate/#bg/qfield/109478893

File: i18n/qfield_bg.ts [hunk 1]
```diff
@@ -0,0 +1,1042 @@
...
+        <source>QGIS projects (*.qgs *.qgz)</source>
+        <translation>QGIS проекти (*.qgs *.qgz)</translation>
+    </message>
+    <message>
+        <source>You are now in browse mode</source>
+        <translation>Активиран е режим преглед</translation>
+    </message>
+    <message>
+        <source>You are now in digitize mode</source>
+        <translation>Активиран е режим цифроване</translation>
```
Comment (hristo-mavrodiev) [inline i18n/qfield_bg.ts:995]: Probably better translation is :
Активиран е режим редактиране

File: i18n/qfield_bg.ts [hunk 2]
```diff
@@ -0,0 +1,1042 @@
...
+        <source>Positioning turned off</source>
+        <translation>Позиционирането е изключено</translation>
+    </message>
+    <message>
+        <source>Main Menu</source>
+        <translation>Главно меню</translation>
+    </message>
+    <message>
+        <source>Mode</source>
+        <translation type="vanished">Прелистване</translation>
```
Comment (hristo-mavrodiev) [inline i18n/qfield_bg.ts:859]: Mode -> Режим

**Comment by m-kuhn** [comment]
@hristo-mavrodiev would you mind joining the translation team at https://www.transifex.com/opengisch/qfield-for-qgis/ ? That's where the "real" translation is done.

File: i18n/qfield_bg.ts [hunk 3]
```diff
@@ -0,0 +1,1042 @@
...
+    </message>
+    <message>
+        <source>Browse</source>
+        <translation type="vanished">Прелистване</translation>
+    </message>
+    <message>
+        <source>Digitize</source>
+        <translation type="vanished">Цифроване</translation>
```
Comment (hristo-mavrodiev) [inline i18n/qfield_bg.ts:867]: Probably better translation will be: Дигитализиране or Редактиране

File: i18n/qfield_bg.ts [hunk 4]
```diff
@@ -0,0 +1,1042 @@
...
+        <source>Time interval (s)</source>
+        <translation>Времеви интервал (сек.)</translation>
+    </message>
+    <message>
+        <source>Distance (%1)</source>
+        <translation>Разстояние (%1)</translation>
+    </message>
+    <message>
+        <source>Digitize vertex only when both conditions are met</source>
+        <translation>Цифровай точката само когато и двете условия са изпълнени</translation>
```
Comment (hristo-mavrodiev) [inline i18n/qfield_bg.ts:778]: Цифровай ->Дигитализирай

File: i18n/qfield_bg.ts [hunk 5]
```diff
@@ -0,0 +1,1042 @@
...
+        <source>When progressive rendering is enabled, the map will be drawn every 250 milliseconds while rendering.</source>
+        <translation>При включено прогресивно изчертаване, картата ще бъде изчертавана на всеки 250 мс. докато изчертаването приключи.</translation>
+    </message>
+    <message>
+        <source>Show digitizing information</source>
+        <translation>Показване на информация при дигитализиране</translation>
+    </message>
+    <message>
+        <source>When switched on, coordinate information, such as latitude and longitude, is overlayed onto the canvas while digitizing new features or using the measure tool.</source>
+        <translation>Когато евключено, информация за координатите (ширина и дължина) ще се показват върху екрана докато се цифроват нови обекти или се използва инструментът за измерване.</translation>
```
Comment (hristo-mavrodiev) [inline i18n/qfield_bg.ts:625]: цифроват -> дигитализират

**Comment by hristo-mavrodiev** [comment]
ok, sorry for the spam.

**Comment by m-kuhn** [comment]
you are most welcome to join the team :)

File: i18n/qfield_bg.ts (hunk 2 above)
Comment (suricactus) [reply i18n/qfield_bg.ts:859]: This translation is in read-only mode and no longer used. I guess I translated it by mistake. The first release of QField with Bulgarian translations would be 1.6, so ignore those marked `type="vanished"`.

**Comment by suricactus** [comment]
Welcome to the translation team :)

**Pull request merged**